src/Microsoft.VisualStudio.SlowCheetah/Build/Microsoft.VisualStudio.SlowCheetah.targets

```diff
@@ -83,7 +114,32 @@ Copyright (C) Sayed Ibrahim Hashimi, Chuck England 2011-2013. All rights reserve
     <SlowCheetah.TransformTask Source="@(ScFilesToTransform->'%(SourceFile)')"
                               Transform="%(TransformFile)"
                               Destination="%(DestinationFile)"
-                              Condition=" Exists('%(TransformFile)')"/>
+                              Condition="Exists('%(TransformFile)')"/>
+
+    <ItemGroup>
+      <!--Gather each category of files to transform again since more may have been added between targets-->
+      <_ScNoneFilesToTransform Include="@(ScFilesToTransform)" Condition="('%(BuildAction)' == '' or '%(BuildAction)' == 'None') and Exists('%(DestinationFile)')" />
```

Comment:
This can be simplified to assume that any new items to `ScFilesToTransform` are going to not have a `BuildAction`, and only check for that and add them to `None`. If the user cared about the build action, they would have included it in that item group to begin with.